src/App.jsx

```diff
@@ -1,40 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+  const handleTabSelected = (tab) => {
+    setSelectedTab(tab);
+  };
```

Comment:
there is no need to create this addtional function-container for `setSelectedTab`,

you can just pass `setSelectedTab` to `Tabs` component:
onTabSelected={setSelectedTab}